api/api_ethereum.go

```diff
@@ -610,6 +606,7 @@ func (api *EthereumAPI) Call(ctx context.Context, args EthTransactionArgs, block
 
 	err = blockchain.GetVMerrFromReceiptStatus(status)
 	if err != nil && isReverted(err) && len(result) > 0 {
+		logger.Info("revert error occured", "error", err)
```

Comment:
Revert during Call API is common cases for EN. Is there any reason we print a log here?

Reply:
I did it because I thought it would be good to show it as a log. I will remove it again.